numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -297,6 +298,48 @@ def get_conda_ctk():
     return os.path.dirname(max(paths))
 
 
+def get_wheel_static_lib():
+    try:
+        import nvidia
+
+        nvidia_path = nvidia.__path__[0]
+    except ImportError:
+        return
+
+    cudadevrt_path = os.path.join(nvidia_path, "cuda_runtime", "lib", "x64")
+
+    if os.path.exists(os.path.join(cudadevrt_path, "cudadevrt.lib")):
+        return cudadevrt_path
+
+
+def get_wheel_include():
+    try:
+        import nvidia
+
+        nvidia_path = nvidia.__path__[0]
+    except ImportError:
+        return
+
+    cuda_include_path = os.path.join(nvidia_path, "cuda_runtime", "include")
```

Comment:
This is the cuda 12 path, for CUDA 13 need to use: `os.path.join(nvidia_path, "cu13", "include")`